json-jackson/shared/src/main/scala/rapture/json-jackson/ast.scala

```diff
@@ -22,7 +22,7 @@ import rapture.data.DataTypes
 import com.fasterxml.jackson.databind._
 import com.fasterxml.jackson.databind.node.NullNode
 
-import scala.collection.JavaConversions._
+import scala.collection.JavaConverters._
```

Comment:
This change is just to force the conversions to be explicit, right? I prefer that!